app/views/users/index.html.erb

```diff
@@ -1,2 +1,4 @@
-<h1>All Users</h1>
-<p>Here is a list of all users</p>
+<% @users.each do |user| %>
+  <%= render partial: 'shared/user_layout', locals: {user: user} %>
```

Comment:
- It's very nice that you have used `partial` so that you'll avoid using the same code again. :+1: :1st_place_medal: